### ISSUE TITLE
loss function for dragonnet

### ISSUE THREAD

**Issue by kumar4372** [Zhuang-Zhuang-Liu/DeepUplift#3]
Is targeted regularization correct in dragonnet_loss?

This implementation looks more correct:
https://github.com/farazmah/dragonnet-pytorch/blob/master/dragonnet/model.py

**Comment by Zhuang-Zhuang-Liu** [comment]
Thanks for your issue. The bug has been fixed.
